syntax/test_helpers.go

```diff
@@ -59,19 +59,16 @@ func AssertCodeEvalsToType(t *testing.T, expected interface{}, code string) bool
 	return true
 }
 
-// AssertCodePanics asserts that the code panics when executed.
+// AssertCodePanics asserts that code panics when executed.
 func AssertCodePanics(t *testing.T, code string) bool {
-	pc := ParseContext{SourceDir: ".."}
-	ast, err := pc.Parse(parser.NewScanner(code))
-	if !assert.NoError(t, err, "parsing code: %s", code) {
-		return false
-	}
-	codeExpr := pc.CompileExpr(ast)
-	if !rel.AssertExprPanics(t, codeExpr) {
-		t.Logf("code:     %s", code)
-		return false
-	}
-	return true
+	return assert.Panics(t, func() {
+		pc := ParseContext{SourceDir: ".."}
+		ast, err := pc.Parse(parser.NewScanner(code))
+		if assert.NoError(t, err, "parsing code: %s", code) {
+			codeExpr := pc.CompileExpr(ast)
+			codeExpr.Eval(rel.EmptyScope) //nolint:errcheck
+		}
```

Comment:
you can use `require.NoError` instead.

Reply:
to replace `assert.NoError`

Reply:
That fails the entire test. I want to allow the test to continue after this assertion fails.